projects/adin1300_dual/zc706/system_bd.tcl

```diff
@@ -0,0 +1,24 @@
+
+source $ad_hdl_dir/projects/common/zc706/zc706_system_bd.tcl
+
+source ../common/adin1300_dual_bd.tcl
+
+ad_ip_parameter clk_wiz CONFIG.CLKOUT1_REQUESTED_OUT_FREQ 200
+ad_ip_parameter clk_wiz CONFIG.MMCM_DIVCLK_DIVIDE 1
+ad_ip_parameter clk_wiz CONFIG.MMCM_CLKFBOUT_MULT_F 8
+ad_ip_parameter clk_wiz CONFIG.MMCM_CLKIN2_PERIOD 10.0
+ad_ip_parameter clk_wiz CONFIG.MMCM_CLKOUT0_DIVIDE_F 5
+ad_ip_parameter clk_wiz CONFIG.CLKOUT1_JITTER 109.241
```

Comment:
Do we need all parameters for this ? I think some of them should not be instantiated explicitly